src/exchanges/bybit/account-events/log-account-events-usdt.ts

```diff
@@ -42,5 +37,12 @@ wsClient.on('reconnected', (data) => {
   console.log('ws has reconnected ', data?.wsKey);
 });
 
-// subscribe to private endpoints
-wsClient.subscribe(['position', 'execution', 'order', 'wallet']);
+// (v5) subscribe to multiple topics at once
+wsClient.subscribeV5(
+  ['orderbook.50.BTCUSDT', 'orderbook.50.ETHUSDT'],
+  'linear',
+);
+
+// (v5) and/or subscribe to individual topics on demand
+wsClient.subscribeV5('position', 'linear');
+wsClient.subscribeV5('publicTrade.BTC', 'option');
```

Comment:
Might be confusing to have a random options topic, maybe this should be on linear too?

Reply:
Removed, accidetnally copied from bybit repo haha